neo/NeoSystem.cs

```diff
@@ -30,9 +31,9 @@ public class NeoSystem : IDisposable
         private ChannelsConfig start_message = null;
         private bool suspend = false;
 
-        public NeoSystem(Store store)
+        public NeoSystem(Store store = null)
         {
-            this.store = store;
+            this.store = store ?? Plugin.Storages.First().GetStore();
```

Comment:
Is this deterministic? If the user shuts the node down, installs a plugin and starts it back up, what's the expected behavior?

Reply:
Now we can load storage plugin by its name.